pagespeed/envoy/envoy_fetch.h

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- * 
+ *
```

Comment:
Can be back out the reformatting? It makes it harder to figure out what changed.
It's fine to reformat before this lands, but it's better to do so either before or after review.